Rakefile

```diff
@@ -1,7 +1,7 @@
 require "logstash/devutils/rake"
 
 desc "Get jars"
-task :get_jars do
+task :vendor do
```

Comment:
Hmm, we already have a `rake vendor` task in devutils that fetches files for us. I won't necessarily require a refactor to use that method here, but was there a reason it wasn't used?